backend/module/chaos/serverexperimentation/xds/xds_cache.go

```diff
@@ -27,7 +26,7 @@ func PeriodicallyRefreshCache(s *Server) {
 }
 
 func refreshCache(ctx context.Context, storer experimentstore.Storer, snapshotCache gcpCacheV3.SnapshotCache, ttl *time.Duration, rtdsConfig *RTDSConfig, ecdsConfig *ECDSConfig, scope tally.Scope, logger *zap.SugaredLogger) {
-	allRunningExperiments, err := storer.GetExperiments(ctx, "type.googleapis.com/clutch.chaos.serverexperimentation.v1.HTTPFaultConfig", experimentation.GetExperimentsRequest_STATUS_RUNNING)
+	allRunningExperiments, err := storer.GetExperiments(ctx, "", experimentation.GetExperimentsRequest_STATUS_RUNNING)
```

Comment:
Is this right? Does GetExperiments treat empty list as any experiment?

Reply:
Yes it treats the empty string as any experiment since the SQL query ends up being ` AND ($1 = '' OR $1 = experiment_config.details ->> '@type')` where $1 is equal to ''